cfme/fixtures/candu.py

```diff
@@ -1,15 +1,48 @@
 """
 Fixtures for Capacity and Utilization
 """
+import math
+from collections import namedtuple
+from datetime import date
+
 import fauxfactory
 import pytest
+from wrapanapi import VmState
 
+import cfme.intelligence.chargeback.assignments as cb
```

Comment:
I'm not really a fan of just using `cb` for this import as its not clear what it is, but `cb_ass` doesn't seem too appropriate.

Reply:
Since this is only used for two classes, I think we should just import those classes specifically. `StorageAssign` and `ComputeAssign`